libs/stack/stack-ui/src/components/fields/TextArea/textArea.stories.jsx

```
@@ -0,0 +1,54 @@
+}
+
+export const Disabled = {
+  render: Template.bind({}),
+  name: 'Disabled',
+
+  args: {
+    disabled: true,
+  },
+}
+
+export const Error = {
```

Biome

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)